Index Error

E                       KeyError: 'Indexing with a boolean dask array is not allowed. This will result in a dask array of unknown shape. Such arrays are unsupported by Xarray.Please compute the indexer first using .compute()'

See precommit PR

--- a/xskillscore/core/probabilistic.py
+++ b/xskillscore/core/probabilistic.py
@@ -1149,7 +1149,7 @@ def _drop_intermediate(fpr, tpr):
             fpr.isel(probability_bin=0, drop=False).astype("bool"),
         ],
         "probability_bin",
-    )
+    ).compute()
     optimal_idxs["probability_bin"] = np.arange(optimal_idxs.probability_bin.size)
     if isinstance(optimal_idxs, xr.Dataset):
         optimal_idxs = optimal_idxs.to_array()

